creator-node/Dockerfile

```diff
@@ -5,9 +5,40 @@ WORKDIR /app
 COPY package*.json ./
 RUN npm install --loglevel verbose
 
-FROM node:14.16-alpine
```

Comment:
is the intention here to reduce image size?

Reply:
It's mostly so that we have finer control over our dependencies on this image